### PR TITLE
docs: Added Motasim's data

### PR DESCRIPTION
**Pull request by motasimmakki**

### CODE REVIEW

File: public/data/motasimmakki.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "icon": "linkedin"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/_motasim_",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (Panquesito7) [inline public/data/motasimmakki.json:23]:
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/motasimmakki.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://www.linkedin.com/in/motasim-010b0a135/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/_motasim_",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/motasimmakki.json:22]:
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by eddiejaoude — APPROVED** [review]
Great collaboration 👍

**Comment by motasimmakki** [comment]
@eddiejaoude Kindly invite me to join the org. ✨

**Comment by krishguptadev** [comment]
Open a issue on our support repo to get invited: https://github.com/EddieHubCommunity/support/issues/new/choose

**Pull request merged**